src/transformers/models/mamba/modeling_mamba.py

```diff
@@ -261,6 +271,9 @@ def slow_forward(self, input_states, cache_params: Optional[MambaCache]=None, ca
             )
             hidden_states = self.act(self.conv1d(hidden_states)[..., :seq_len])         # [batch, intermediate_size, seq_len]
 
+        if attention_mask is not None:
+            hidden_states = hidden_states * attention_mask.unsqueeze(1)
```

Comment:
valid for all of these operations: no `dtype` issue encountered here when the model is loaded in different precisions?

Reply:
In principle no since attention mask is a `torch.LongTensor`, in the slow test the model is loaded in bf16 and there is no issue !